ports/linux/bip-init.c

```diff
@@ -508,6 +525,28 @@ int bip_get_local_address_ioctl(char *ifname, struct in_addr *addr, int request)
     return rv;
 }
 
+/**
+ * @brief Issue a specific request foor an interface via an ioctl() call.
+ * @param ifname - the interface name
+ * @param flags [out] the active flag word of the device.
+ * @param request - the ioctl() request
+ * @return 0 on success, else the error from the ioctl() call.
+ */
+int bip_get_flags_ioctl(char *ifname, short *flags, int request)
+{
+    struct ifreq ifr = { { { 0 } }, { { 0 } } };
+    struct sockaddr_in *tcpip_address;
```

Comment:
Where is this used?  If not, please remove it.

Reply:
Fix